pkg/apis/ceph.rook.io/v1/nfs.go

```diff
@@ -51,19 +50,19 @@ func (n *CephNFS) IsHostNetwork(c *ClusterSpec) bool {
 	return c.Network.IsHost()
 }
 
-func (n *CephNFS) ValidateCreate() (admission.Warnings, error) {
+func (n *CephNFS) ValidateCreate() (error, error) {
```

Comment:
We just need to return one error, right?
```suggestion
func (n *CephNFS) ValidateCreate() error {
```

Reply:
Actually, do we even need these `Validate` methods? Or can we remove them completely?

Reply:
`Validate()` in general also called from respective `controller`

Reply:
Also, I guess `golangci` lint should give error when found `unused` var/method

Reply:
The linter only flags private methods that are not used. Public methods (upper case) are not flagged since something outside the project might be using it.